index.src.html

```diff
@@ -1815,6 +1825,12 @@
     {{HardwareAcceleration}}.
   </dd>
 
+<dt><dfn dict-member for=VideoEncoderConfig>alpha</dfn></dt>
+  <dd>
+    Whether the alpha component of the {{VideoFrame}} inputs should be kept
+    or discarded prior to encoding.
```

Comment:
Is it stated somewhere how the encoder should behave when it gets a video frame with alpha option of either "keep" or "discard"?
The intuitive behavior (for me) says that "discard" in the encoder overrules a "keep" on the video frame, but this might need to be said explicitly somewhere.

Reply:
You've got the right idea. @tguilbert-google, wdyt of adding more clarifying detail?

Reply:
I've added a clarifying sentence. Thanks for pointing this out!